### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Edit Configuration popup_Keyboard: If any of the dropdowns “Job upload storage, Advanced configuration” is in expanded state then focus is not visible for 3 tabs else focus is not visible for 2 tabs before going to “spark run job” button.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3602]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.4.3; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.4.3

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA.
3.Tab till Create New Project button and press enter
4.Tab till Azure Spark/HDInsight-->Spark Project Scala
5.Tab till Build Tools-->Maven
6.Tab till Next button and press enter
7.Tab till finish button and press enter
8.Navigate to project name-->src-->main-->scala-->project name and open the context menu using keyboard key
9.Tab till Submit Spark Application and press enter.
10.Tab till “Referenced files” field.
11.Tab once and verify whether focus is pointing to the next control without any focus order issue or not.

**Actual Result:**
If any of the dropdowns “Job upload storage, Advanced configuration” is in expanded state then focus is not visible for 3 tabs else focus is not visible for 2 tabs before going to “spark run job” button.

**Expected Result:**
When focus is on “Referenced files” field, If we press tab focus should immediately point to the next control present in the page without pointing to the controls which are not visible to the users before going to “Spark run job” button.

**User Impact:**
Keyboard users will face difficulty in knowing where the focus is pointing, If the focus is not visible for 2-3 tabs before going to “spark run job” button.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_4_Focus order issue after referenced files field in Edit config popup.pptx](https://github.com/microsoft/azure-tools-for-java/files/3642160/Bug_4_Focus.order.issue.after.referenced.files.field.in.Edit.config.popup.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `Referenced Files`, tab twice, the focus is not visible. Tab three times, focus go to `SparkJobRun`.

![image](https://user-images.githubusercontent.com/32627233/65659500-70787180-e05e-11e9-9631-ca6af0f525e4.png)

**Comment by t-rufang** [comment]
After debug, we find the focus order is as the following picture.

![image](https://user-images.githubusercontent.com/32627233/67008986-3c541600-f11d-11e9-9199-5be62a21b505.png)

Two issues here. When focus is at `uploadPathTextField` and `myWarningLabel`, the focus is invisible. 
1. Here the `uploadPathTextField` is introduced by our toolkit code and we will fix this focus issue.
2. For `myWarningLabel`, the code for this component is controlled by IntelliJ and should be owned by IntelliJ team. Please find another similar issue in the `Application` run configuration, which is wrote by IntelliJ team. In the gif below, when the focus is at the checkbox `Enable capturing form snapshots`, press tab, the focus is invisible(actually it's at the `myWarningLabel`), tab again, focus goes to `OK` button.
![issue_3602](https://user-images.githubusercontent.com/32627233/67009740-dcf70580-f11e-11e9-9977-ede9291a9c1e.gif)

**Comment by t-rufang** [comment]
For the first `uploadPathTextField` focus issue, it's fixed in #3668

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749705
